src/main/java/com/google/jenkins/plugins/computeengine/ComputeEngineInstance.java

```diff
@@ -36,7 +35,7 @@
     public final String sshUser;
     public transient final Optional<WindowsConfiguration> windowsConfig;
     public Integer launchTimeout; // Seconds
-    private Boolean connected;
+    public Boolean connected;
```

Comment:
Could you please add a TODO comment here linking to this issue: https://issues.jenkins-ci.org/browse/JENKINS-55518